packages/apm/src/integrations/tracing.ts

```diff
@@ -542,6 +536,11 @@ export class Tracing implements Integration {
         const startTime = Tracing._msToSec(entry.startTime as number);
         const duration = Tracing._msToSec(entry.duration as number);
 
+        if (transactionSpan.op === 'navigation' && timeOrigin + startTime < transactionSpan.startTimestamp) {
+          logger.log('[Tracing] Discarded performance entry because of navigation');
```

Comment:
Good point, I think we can remove this.
For now, my premise was, since this integration is anyway very beta I thought the more output the better.

Reply:
```suggestion
          logger.log('[Tracing] Discarded performance entry because of navigation', entry);
```

Could we do that? If we discard an entry, will be good to know what it was instead of just seeing a number of log messages that are equal in the console.

Reply:
Talked outside of GH and decided to remove it for now.

We can always add it back if needed.